scripts/rancher/cluster-actions.sh

```diff
@@ -3,125 +3,84 @@
 
 #######################################
 # List clusters managed by Rancher
-# Arguments:
-#   Rancher URL
-#   token
-# Examples:
-#   rancher_list_clusters rancher.random_string.geek xxxxx
 #######################################
 rancher_list_clusters() {
-  local rancherUrl=$1
-  local token=$2
-
   echo "Listing clusters registered in Rancher..."
-  curl -s -k "$rancherUrl/v3/clusters" -H "Authorization: Bearer $token" | jq .
+  kubectl get clusters.provisioning.cattle.io --all-namespaces | awk 'NR>1 {print $2}'
```

Comment:
Shall we standardize on either `jq` or `-o=jsonpath=`?

Reply:
Yes, could we use jq instead for this call?
From what I have seen it's more a standard while working with json data.

Reply:
jq is a standard when it comes to json response, but when working with kubectl output - jsonpath might be better. No need for additional binary, less piping. I've been bitten in a rear for using Rancher's API so many times, that I want to avoid it at all cost. Rancher is all CRD based, we should embrace it and not use API on top of API, change k8s resources directly...

Reply:
Good to have you your feedback on the API.
In this line of code, can we replace awk by jq or jsonpath?
(`kubectl get clusters.provisioning.cattle.io --all-namespaces -o json | jq -r '.items[] | .metadata.name'
` or `kubectl get clusters.provisioning.cattle.io --all-namespaces -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'`)

Reply:
I can agree with the sentiment for not doing API calls and doing everything through CRDs.